docs/developer-docs/setup/playground.md

```diff
@@ -0,0 +1,279 @@
+
+![Candid UI 2](./_attachments/motoko-playground8.png)
+
+To test this method, enter the text 'Welcome to Motoko playground!' in the input box, then select the 'QUERY' button.
+
+![Candid UI 3](./_attachments/motoko-playground9.png)
+
+In the output log, you will see that the canister returns this string of text to you.
+
+![Candid UI 4](./_attachments/motoko-playground10.png)
+
+Next, let's take a closer look at the left side bar menu in the window. This menu has three categories: files, packages, and canisters.
+
+![Left menu](./_attachments/motoko-playground11.png)
+
+To add new files to the Motoko playground workspace, select the `+` button next to the 'FILES' option.
+
+![Add files](./_attachments/motoko-playground12.png)
+
+Then give the new file a file name.
+
+![Create file name](./_attachments/motoko-playground13.png)
+
+To add new packages to the Motoko playground workspace, select the `+` button next to the 'PACKAGES' option.
+
+![Add packages](./_attachments/motoko-playground14.png)
+
+From this menu, you can select from the list of pre-imported Vessel packages, or you can input your own from Github.
+
+![Packages](./_attachments/motoko-playground15.png)
+
+Lastly, the 'CANISTERS' portion of the menu showcases your currently running canister(s) and shows the time remaining for the canister's deployment. When this timer runs out, the canister is automatically uninstalled.
+
+To add another canister, select the `+` button next to the 'CANISTERS' option.
+
+![Add canisters](./_attachments/motoko-playground16.png)
+
+From the 'Add a canister' menu, you can import a canister using the canister's ID, as shown below.
+
+![Import by ID](./_attachments/motoko-playground17.png)
+
+Alternatively, you can deploy a canister using the canister's wasm and .did files. 
+
+![Import by wasm](./_attachments/motoko-playground18.png)
+
+Lastly, let's take a look at the two buttons in the top right corner of the Motoko playground UI: the 'Save and & Share' button and the 'Open Example' button.
+
+The 'Save and Share' button can be used to save your canister's code and share it using a live canister URL. Select this button to demonstrate this workflow.
+
+![Save and share](./_attachments/motoko-playground19.png)
+
+In the log, your canister's URL will be returned for you to share with others:
+
+![Canister URL](./_attachments/motoko-playground20.png)
+
+Then, the 'Open Example' button can be used to open another pre-configured example, or import your own from Github.
+
+![Open example](./_attachments/motoko-playground21.png)
+
+This menu will have the same examples and options that the initial welcome menu contained.
+
+![Examples](./_attachments/motoko-playground22.png)
+
+## Running Motoko playground locally
```

Comment:
IDK how useful this section is. I don't think anyone will want to do this unless they want to work on the playground itself, in which case I'd simply point them to https://github.com/dfinity/motoko-playground